src/main/java/lotto/view/OutputView.java

```diff
@@ -32,9 +34,20 @@ public static void printLottoResult(LottoResult lottoResult, double profitRatio)
     }
 
     private static void printLottoStatistics(LottoResult lottoResult) {
-        for (int i = MIN_WINNING_NUM; i < PRIZE_MONEY.length; i++) {
-            System.out.printf(OUT_MESSAGE_LOTTO_RESULT, i, PRIZE_MONEY[i], lottoResult.getLottoResult(i));
+        Arrays.stream(Rank.values())
+                .sorted(Comparator.comparingInt(Rank::getWinningMoney))
+                .filter(rank -> rank.getCountOfMatch() > 2)
```

Comment:
```suggestion
                .filter(rank -> rank.is적절한네이밍())
```

데이터를 가지고 있는 객체한테 메시지를 전달해보는것도 좋겠네요 !

Reply:
앗.. 수정하겠습니다!